components/script/dom/servoparser/mod.rs

```diff
@@ -102,6 +102,10 @@ enum LastChunkState {
 }
 
 impl ServoParser {
+    pub fn parser_is_not_active(&self) -> bool {
+        self.tokenizer.try_borrow_mut().is_ok()
```

Comment:
Let's add `|| self.can_write()`, so that this is handled correctly when https://github.com/servo/servo/issues/19393 is fixed.